less/_components/quicksilver/dark-theme.less

```diff
@@ -0,0 +1,13 @@
+.prefers-color-scheme{
+    @media (prefers-color-scheme: dark) {  
+        body footer{
+            background-color: @body-dark-theme;
+            color: @text-dark-theme;
+        }
+        
+        body a{
```

Comment:
I don't think we want to do that, this will set all our links to @brand-primary; which would include the menu.

Maybe we only change the link color in `<main>`?

Reply:
@chrisguindon I will implement that in my commit too.